### DISCUSSION TITLE
Python Flask cannot be installed

### DISCUSSION THREAD

**Discussion by Fairies-Feast** [vercel/community#5771] [Help]
### Summary

I created a deployment for Fairies-Feast/amongus-audio, and set the build command to "python main.py" and set the install command to "pip install flask". When I look at the logs, it shows flask being installed: `Successfully installed Jinja2-3.1.3 MarkupSafe-2.1.5 Werkzeug-3.0.1 blinker-1.7.0 click-8.1.7 flask-3.0.2 importlib-metadata-7.0.1 itsdangerous-2.1.2 zipp-3.17.0`, but then it shows:
```bash
[notice] A new release of pip is available: 23.3.1 -> 24.0
--
20:31:13.899 | [notice] To update, run: pip install --upgrade pip
20:31:14.034 | Traceback (most recent call last):
20:31:14.034 | File "main.py", line 1, in <module>
20:31:14.034 | from flask import Flask, render_template
20:31:14.034 | ImportError: No module named flask
20:31:14.039 | Error: Command "python main.py" exited with 1
```

### Steps to Reproduce

1. Create a deployment for Fairies-Feast/amongus-audio, set the build command to "python main.py", set the output dir to ".", install command to "pip install flask"
2. Press Deploy
3. Error will happen during building

**Comment by mikebionic** [comment]
Are you working in a virtual environment? Of not then better to get all packages installed there.
Afterwards do deplyment throug the virtualenv you installed, use the python executable from it.
If it didn't work there might be issue that you're soing build from uset/computer that didn't access the env or where packets weren't istalled..

**Reply by Fairies-Feast** [reply]
What virtual environment? Is that a requirement for Vercel?
